build.sbt

```diff
@@ -79,7 +79,7 @@ lazy val `circe-yaml-scalayaml` = crossProject(JSPlatform, JVMPlatform, NativePl
   .settings(
     description := "Library for converting between scala-yaml AST and circe's AST",
     libraryDependencies ++= Seq(
-      "org.virtuslab" %%% "scala-yaml" % "0.1.0",
```

Comment:
Is this a binary breaking change?

Reply:
@lbialy was this a breaking change? Would downstream user's compilation fail after this upgrade?

Reply:
Less worried about compilation as i am about accidental method not allowed exceptions etc etc

Reply:
I think you should treat it as binary breaking (as in [early semver](https://www.scala-lang.org/blog/2021/02/16/preventing-version-conflicts-with-versionscheme.html#early-semver-and-sbt-version-policy)) because `Decoder` derivation was rewritten from inlines to full macros. I think [this](https://github.com/VirtusLab/scala-yaml/compare/v0.1.0...v0.3.0#diff-32973d256749bcd5d6f523448e86986a9ce9483feb6047a712ae41b7e23d2670L31) change (for example) will cause issues for any inlined code from 0.1.0 as this method no longer exists on the DecoderMacros object.

Reply:
lets bump the tlBaseVersion then.

Reply:
done